MITLicense.md

```diff
@@ -0,0 +1,8 @@
+The MIT License (MIT)
```

Comment:
This isn't actually Markdown. Recommend switching file format to txt or using Markdown version - https://github.com/IQAndreas/markdown-licenses/blob/master/mit.md